docs/docs/blocks/blocks-in-forms.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Blocks in forms
+sidebar_position: 6
+---
+
+# Using blocks in forms
+
+This guide describes how to use a block in a Final Form.
+We use a product form with a `DamImageBlock` as an example.
+The product has the following schema in GraphQL:
+
+```graphql
+type Product {
+    id: ID!
+    title: String!
+    // highlight-next-line
+    image: DamImageBlockData!
+}
+```
+
+The following steps are necessary to add the `image` field to the form:
+
+1. Create a Final Form component for the block
+2. Create the block state when initializing the form
+3. Create the block output when submitting the form
+
+:::info
+
+The block transitions between the various states when being used in a form.
+The states and transitions can be viewed [here](lifecycle#the-block-in-the-admin).
```

Comment:
the link is not working, maybe you need ./livecycle?

Reply:
For some reason it worked locally. Fixed: b5e8fd5f22f49d3f8705c31942ec6b2a18292de6